space2stats_api/src/space2stats/h3_utils.py

```diff
@@ -1,67 +1,122 @@
-from itertools import chain
-from typing import Any, Dict, List, Optional
+from typing import Any, Dict, Iterable, List, Optional, Set
 
 import h3
 from shapely.geometry import MultiPolygon, Point, Polygon, mapping, shape
 
+# https://h3geo.org/docs/core-library/restable
+MAX_RESOLUTION = 15
+
+
+def _get_parent(
+    h3_ids: Iterable[str], resolution: int, target_resolution: int
+) -> Set[str]:
+    if resolution == target_resolution:
+        return set(h3_ids)
+
+    assert resolution > target_resolution
```

Comment:
maybe add an error message here 

```python
assert resolution > target_resolution, "error message that tells why"
```

Reply:
We can actually remove the `_get_parent` function!